plugins/modules/firewalld.py

```diff
@@ -636,10 +696,12 @@ def main():
             interface=dict(type='str'),
             masquerade=dict(type='str'),
             offline=dict(type='bool'),
+            target=dict(type='str', required=False, choices=['default', 'ACCEPT', 'DROP', 'REJECT']),
```

Comment:
I think by default `required` is `False` and there is no need to state that.
```suggestion
            target=dict(type='str', choices=['default', 'ACCEPT', 'DROP', 'REJECT']),
```

Reply:
Done